core/src/main/java/com/mygdx/auber/entities/Prisoners.java

```diff
@@ -62,4 +72,35 @@ public static void render(Batch batch) {
             }
         }
     }
+
+    // TODO: Understand why prisoners is currently nothing?
+    public static String encode() {
+        Array<Vector2> prisonerLocations = new Array<>();
+        System.out.print("Encoding prisoners.");
+        System.out.print(Prisoners.prisoners.size);
+
+        for (Sprite sprite : Prisoners.prisoners) {
+            prisonerLocations.add(new Vector2(sprite.getX(), sprite.getY()));
+        }
+        return prisonerLocations.toString();
+    }
+
+    // TODO: Needs testing.
+    public static void LoadFromString(String value1) {
+        String value = "[(1936.0,3632.0), (1968.0,3632.0), (2032.0,3632.0), (2064.0,3632.0), (2128.0,3632.0), (2160.0,3632.0)]";
```

Comment:
Is this meant to be JSON or similar?  Could try using a JSON library if so.

Reply:
Prisoners isn't currently finished right now - it literally does nothing, i'm gonna fix that tomorrow.

Reply:
There are no prisoners visible for anyone at the moment, there hasn't been for a while so you're good.